lib/histogram.js

```diff
@@ -0,0 +1,9 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
```

Comment:
@mdtusz could you make the glob [here](https://github.com/plotly/plotly.js/blob/master/tasks/header.js#L36) grab the `lib/` files too before we forget.